components/FeaturedPost.tsx

```diff
@@ -1,11 +1,14 @@
 import Link from 'next/link';
 
-const FeaturedPost = () => {
-  return (
-    <div className="grid lg:grid-cols-2 grid-cols-1 items-center mt-12 md:-mx-12 lg:gap-x-20 gap-y-10 lg:gap-y-0">
+const FeaturedPost = ({ posts }) => {
+  return posts.slice(0, 1).map(({ title, author, desc }) => (
```

Comment:
related to my previous comment, we would no longer need to `slice()` and `map()` the items here because it only receives 1 post